### PR TITLE
MIG528: DVM and missing node selectors to Troubleshooting (1.4.1)

### PR DESCRIPTION
**Pull request by apinnick**
https://issues.redhat.com/browse/MIG-528

### CODE REVIEW

**Review by sergiordlr — APPROVED** [review]
Looks good to me.

Thank you very much!

File: modules/migration-dvm-error-node-selectors.adoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+// * migration/migrating_3_4/troubleshooting-3-4.adoc
+// * migration/migrating_4_1_4/troubleshooting-4-1-4.adoc
+// * migration/migrating_4_2_4/troubleshooting-4-2-4.adoc
+
+[id="migration-dvm-error-node-selectors_{context}"]
+= Direct volume migration does not complete
+
+If direct volume migration does not complete, the target cluster might not have the same `node-selector` annotations as the source cluster.
+
+{mtc-short} migrates namespaces with all annotations in order to preserve security context constraints and scheduling requirements. During direct volume migration, {mtc-short} creates Rsync transfer pods on the target cluster in the namespaces that were migrated from the source cluster. If a target cluster namespace does not have the same annotations as the source cluster namespace, the Rsync transfer pods cannot be scheduled. They remain in a `Pending` state.
```
Comment (adellape) [inline modules/migration-dvm-error-node-selectors.adoc:11]: Should `{mtc-short}` here be `{mtc-full} ({mtc-short})`? I noticed some of the modules in this assembly started off spelling it out for the first instance, but not all.

Comment (adellape) [reply modules/migration-dvm-error-node-selectors.adoc:11]: s/in order to/to/

Comment (adellape) [reply modules/migration-dvm-error-node-selectors.adoc:11]: s/They/The pods/

File: modules/migration-dvm-error-node-selectors.adoc [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+You can identify and fix this issue by performing the following procedure.
+
+.Procedure
+
+. Check the status of the `MigMigration` pod:
++
+[source,terminal]
+----
+$ oc describe migmigration <88435fe0-c9f8-11e9-85e6-5d593ce65e10> -n openshift-migration <1>
```
Comment (adellape) [inline modules/migration-dvm-error-node-selectors.adoc:21]: Could the `<88435fe0-c9f8-11e9-85e6-5d593ce65e10>` replaceable be more of a description than an example itself? E.g., `oc describe migmigration <pod_name>`?

**Review by adellape — COMMENTED** [review]
Just some minor questions! :+1:

File: modules/migration-dvm-error-node-selectors.adoc [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+If direct volume migration does not complete, the target cluster might not have the same `node-selector` annotations as the source cluster.
+
+{mtc-short} migrates namespaces with all annotations in order to preserve security context constraints and scheduling requirements. During direct volume migration, {mtc-short} creates Rsync transfer pods on the target cluster in the namespaces that were migrated from the source cluster. If a target cluster namespace does not have the same annotations as the source cluster namespace, the Rsync transfer pods cannot be scheduled. They remain in a `Pending` state.
+
+You can identify and fix this issue by performing the following procedure.
+
+.Procedure
+
+. Check the status of the `MigMigration` pod:
++
```
Comment (hu-weihua) [inline modules/migration-dvm-error-node-selectors.adoc:18]: `MigMigration` should be a CRD, not a pod

File: modules/migration-dvm-error-node-selectors.adoc [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+[source,terminal]
+----
+$ oc describe migmigration <88435fe0-c9f8-11e9-85e6-5d593ce65e10> -n openshift-migration <1>
+----
+<1> Specify the `MigMigration` pod.
```
Comment (hu-weihua) [inline modules/migration-dvm-error-node-selectors.adoc:23]: I guess it should be **<1> Specify the `MigMigration` `NAME`**.  
Below is the output of `oc get migmigration` commond :
```
$ oc get migmigration
NAME                                        READY   PLAN                                            STAGE   ROLLBACK   ITINERARY   PHASE                    AGE
ocp-24659-mysql-mig-1612517241                      ocp-24659-mysql-migplan-1612517241              false              Final       Completed                3h20m

```

**Pull request merged**